google/cloud/monitoring_v3/_dataframe.py

```diff
@@ -94,9 +99,7 @@ def _build_dataframe(time_series_iterable, label=None, labels=None):  # pragma:
     for time_series in time_series_iterable:
         pandas_series = pandas.Series(
             data=[_extract_value(point.value) for point in time_series.points],
-            index=[
-                point.interval.end_time.ToNanoseconds() for point in time_series.points
```

Comment:
Ah what this seems to do is convert the time to just nanoseconds... https://proto-plus-python.readthedocs.io/en/latest/reference/datetime_helpers.html#proto.datetime_helpers.DatetimeWithNanoseconds